apps/web/lib/getTemporaryOrgRedirect.ts

```diff
@@ -45,7 +45,9 @@ export const getTemporaryOrgRedirect = async ({
 
   // Use the first redirect origin as the new origin as we aren't supposed to handle different org usernames in a group
   const newOrigin = new URL(redirects[0].toUrl).origin;
-  const query = currentQueryString ? `?${currentQueryString}` : "";
+  const query = currentQueryString
```

Comment:
When doing the org redirect add this param to convey that the page has been reached due to this reason.